packages/lumx-react/src/components/tabs/TabList.tsx

```diff
@@ -80,12 +88,10 @@ const Tabs: React.FC<TabsProps> = ({
             })}
         >
             <div className={`${CLASSNAME}__links`}>{tabs}</div>
```

Comment:
A few things are missing a11y-wise to be perfect here.

* The div wrapping the tabs should have the `tablist` role (that's easy :p)
* In a perfect world, it should also have either a `aria-label` or `aria-labelledby` attribute to explain the purpose of the tabs (ex : Settings). I understand we might not want to force users to set them, but it would be nice to be able to when using the component. 
So either we destructure the props "aria-label" or "aria-labelledby" so we can set them on the wrapper (whether at least one of them should be required or not is up to debate), or maybe we could add a "wrapperProps" that is spread here ? 

```suggestion
            <div className={`${CLASSNAME}__links`} role="tablist" aria-label={ariaLabel}>{tabs}</div>
```

Or 

```suggestion
            <div className={`${CLASSNAME}__links`} role="tablist" {...wrapperProps}>{tabs}</div>
```

Reply:
The first suggestion seems good to me. I don't know if we should enforce it but having it as a props can help, even more if the documentation and demo include it.

Reply:
Returning on this, it might actually not be a problem to add the role + label on the parent element (the one that has the forwardedProps). As you wish :)